doc/humor/alices.pdp10

```diff
@@ -171,16 +172,16 @@ hairiest TECO hacker of them all was coming over to me.  And he was
 mean and nasty and horrible and undocumented and all kinds of stuff.
 And he sat down next to me and said:
 
-[1:i*u14<q1&377.f"nir'q1/400.u1>[8
```

Comment:
Are the escape characters here right? (cf AI: DIGEX; MTD ALICE)

Reply:
I took the "original" from AI: HUMOR;.  As far as I can see (but then, it's TECO) it matches the "latest" version except control characters have been updated to ^ and a printable character.  The DIGEX copy is different from both.